README.md

```diff
@@ -1,150 +1,141 @@
+[![Coverage Status](https://coveralls.io/repos/github/deriv-com/deriv-api-docs/badge.svg?branch=master)](https://coveralls.io/github/deriv-com/deriv-api-docs?branch=master)
 
-<!--
-![CircleCI](https://img.shields.io/circleci/build/github/binary-com/deriv-api-docs)
-![Sonar Tech Debt](https://img.shields.io/sonar/tech_debt/binary-com_deriv-app?server=https%3A%2F%2Fsonarcloud.io)
-![Sonar Violations (short format)](https://img.shields.io/sonar/violations/binary-com_deriv-app?server=https%3A%2F%2Fsonarcloud.io) -->
+## Description
 
-**In this document**:
+Welcome to the official API documentation for Deriv. Here, you will find comprehensive guides and documentation to help you start working with our APIs as quickly as possible, as well as support if you get stuck. Let's jump right in!`
 
-- [Other Documents](#other-documents)
-- [Pre-installation](#Pre-installation)
-- [Quick start](#Quick-start)
-- [Packages](#packages)
-- [Working With This Repo](#working-with-this-repo)
-  - [Package names](#package-names)
-  - [Usage](#usage)
-    - [Starting a Development Server](#starting-a-dev-server)
-    - [Release](#release)
-- [PR Guidelines](#pr-guidelines)
+## Quick Access
 
-## Other documents:
+- [Prerequisite](#Prerequisite)
+- [Setup Guide](#Setup-Guide)
+  - [Clone](#clone-the-project)
+  - [Install Dependencies](#install-dependencies)
+  - [Production Build](#build-the-project)
+- [How to Contribute](#How-to-Contribute)
+  - [Fork Repository](#fork-the-project)
+  - [Configure Origin & Upstream](#configure-origin--upstream)
+  - [Branching and Pull Request Guidelines](#Branching-and-Pull-Request-Procedure)
+- [Project Structure](#Project-Structure)
 
-- [General](docs/README.md) - Contains general philosophy and overview of this package
+## Prerequisite
 
-## Pre-installation
+To ensure smooth running and contribution to this project, please make sure you have the following packages installed in your environment:
 
-Before running or contributing to this project, you need to install the following packages for your environment
+| Tool | Version   | Reference                                                  |
+| ---- | --------- | ---------------------------------------------------------- |
+| Node | >=18.20.0 | [Download](https://nodejs.org/en/download/package-manager) |
+| npm  | >=10.7.0  | -                                                          |
+| Git  | -         | [Download](https://git-scm.com/downloads)                  |
 
-- node >=16.16.0
-- npm >=7.21.0
-- docusaurus >=2.0.0
-- @deriv/ui >=0.1.0
-- git (for `contribution`)
-- docker-cli
+**Note**: Ensure that `node -v` and `sudo node -v` show the same version.
 
-**Note**: `node -v` and `sudo node -v` should be the same version.
+## Setup Guide
 
-## Quick start
+### Clone the Project
 
-1.  **Fork the project**
+    git clone git@github.com:[username]/deriv-api-docs.git
```

Comment:
@ahmed-deriv as discussed, let's keep this section as generic clone and let's create another section under fork for cloning from the forked repo.